raptorWeb/templates/raptormc/users/user.html

```diff
@@ -5,7 +5,7 @@
 
 {% block main_block %}
 
-<div hx-get="/profile{{request.path|slice:'5:'}}" hx-trigger="load delay:0.1s" hx-target="#user_box" hx-swap="innerHTML"></div>
+<div hx-get="/profile{{request.path|slice:'5:'}}?requested_user={{request.path|slice:'6:'}}" hx-trigger="load delay:0.1s" hx-target="#user_box" hx-swap="innerHTML"></div>
```

Comment:
The value used for `slice` here needs to be the length of the defined `BASE_USER_URL` such that if the "main" app that is requesting this info can work no matter the length of their own user URL schema.